.gitignore

```diff
@@ -2,4 +2,4 @@ __pycache__/
 *.py[cod]
 *$py.class
 yolov3.weights
-yolov3.cfg
+yolov3.cfg
```

Comment:
I think the original file may have had a newline. We should preserve it.

Reply:
I don't think it changes to have an extra line, but I added it. Let me know if there are other changes to make